src/meta/src/hummock/manager/compaction/compaction_group_schedule.rs

```diff
@@ -118,7 +118,6 @@ impl HummockManager {
 
         let created_tables = if let Some(created_tables) = created_tables {
             // if the created_tables is provided, use it directly, most for test
-            const { assert!(cfg!(debug_assertions)) };
```

Comment:
Might keep the assertion, and use `expect` to suppress the clippy warning.

Reply:
Of course, you can commit directly to the branch.